src/connector.cpp

```diff
@@ -1251,7 +1242,10 @@ bool HostConnector::switchScene(const uint8_t scene)
 
     _current.scene = scene;
 
-    const Host::NonBlockingScope hnbs(_host);
+    // TODO: potentially better to do without fades
```

Comment:
this working now with https://github.com/mod-audio/mod-host/commit/e836641bd21f8dc2a50f04016547c4bf611982a7

we can pass 0 as the arg to params_flush and it wont trigger plugin resets